style.css

```diff
@@ -478,3 +482,86 @@ ul li a {
 .skill {
   margin-left: 10px;
 }
+
+.contact-form {
+  display: flex;
+  height: 100vh;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  background: #fff;
+}
+
+.contact-form p {
+  width: 327px;
+  height: 216px;
+  font-family: "Inter", sans-serif;
+  font-weight: 700;
+  line-height: 44px;
+  text-align: center;
+  font-size: 32px;
+  color: #172b4d;
+  margin: 200px auto;
+  padding-top: 6rem;
+}
+
+form {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+input[type="text"],
+[type="email"] {
+  width: 327px;
+  border: none;
+  outline: none;
+  border-bottom: 1px solid #dbd8d7;
+  padding: 12px 15px;
+  margin: 10px;
+  text-align: left;
+  font-family: "Inter", sans-serif;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+textarea {
+  width: 327px;
+  height: 114px;
+  margin: 20px 8px;
+  padding: 12px 15px;
+  font-family: "Inter", sans-serif;
+  color: #3c3a39;
+  font-size: 15px;
+}
+
+.submit-btn {
+  font-family: "Inter", sans-serif;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
```

Comment:
- [ ] Kindly implement interaction to the button on hover, focus to improve the good user experience of your app.
- 
![image](https://user-images.githubusercontent.com/50941074/159679529-cb96fed8-f5fd-49ff-bbbb-7f5a0936ece1.png)